### PR TITLE
#3607: feature flag extension point search in page editor

### LINKED ISSUE #3607: "Search marketplace" modal for "Button"  is empty/broken
No errors in the console, it's just inactive

https://user-images.githubusercontent.com/1402241/172525520-5974bfd1-fce2-4e83-b068-68f9150768e3.mov

### PR DESCRIPTION
**Pull request by twschiller**
## What does this PR do?

- Closes #3607 
- Puts the "Search Marketplace" button for extension points behind a feature flag in the Page Editor
- Skips the insertion pane for extension points that don't support selectors if the feature flag isn't set
- Shows the unconfigured QuickBar warning at the top of the extension point configuration for quickbars

## Checklist

- [X] Add tests: N/A
- [X] Designate a primary reviewer: @BLoe

### CODE REVIEW

File: src/pageEditor/extensionPoints/quickBar.tsx [hunk 1]
```diff
@@ -223,6 +223,41 @@ function asDynamicElement(element: QuickBarFormState): DynamicDefinition {
   };
 }
 
+export const UnconfiguredQuickBarAlert: React.FunctionComponent = () => {
```
Comment (BLoe) [inline src/pageEditor/extensionPoints/quickBar.tsx:226]: Can we re-use this component in `ActivateBody.tsx` as well?

Comment (twschiller) [reply src/pageEditor/extensionPoints/quickBar.tsx:226]: I considered it originally, but I didn't want to show the paragraph if it was configured. 

Could probably extract it though

Comment (twschiller) [reply src/pageEditor/extensionPoints/quickBar.tsx:226]: 🤦  I want thinking of the quickbar insert pane. You're totally right this overlaps with ActivateBody

File: src/pageEditor/extensionPoints/quickBar.tsx [hunk 2]
```diff
@@ -238,50 +263,39 @@ const config: ElementConfig<undefined, QuickBarFormState> = {
...
-    }, []);
+    const { isConfigured, shortcut } = useQuickbarShortcut();
 
     return (
       <div className="text-center pb-2">
-        {isEmpty(shortcut) ? (
-          <Alert variant="warning">
-            <FontAwesomeIcon icon={faExclamationTriangle} />
-            &nbsp;You have not{" "}
+        {isConfigured ? (
```
Comment (twschiller) [inline src/pageEditor/extensionPoints/quickBar.tsx:270]: Flipped to avoid negated condition

**Review by BLoe — APPROVED** [review]

**Pull request merged**